src/app/components/BuildBanner/index.tsx

```diff
@@ -3,10 +3,12 @@ import { useTranslation } from 'react-i18next'
 import { deploys } from '../../../config'
 import { StickyAlert } from '../StickyAlert'
 
+const useBuildBanners = process.env.REACT_APP_NO_BUILD_BANNERS !== 'true'
```

Comment:
I prefer, `ENABLE` statement flags, so I would do it other way around. But not a big deal. It can stay this way.

Reply:
Yeah I agree. Inverted the naming and semantics.